src/Action.hs

```diff
@@ -531,33 +531,50 @@ moveIdeaToTopic ideaId moveIdea = do
 
 -- * Vote Handling
 
+delegatedOperationOnIdea
```

Comment:
arguments:

1. get the delegate value (together with an operation result that is ignored).  rename to `getDelegate`.
1. run a new operation with delegate and delegatee on
1. this idea.

delegatee is current user.